frontends/bmd/src/utils/ScreenReader/kiosk_text_to_speech.ts

```diff
@@ -0,0 +1,112 @@
+import { assert } from '@votingworks/utils';
+import { SpeakOptions, TextToSpeech } from '../../config/types';
+
+// Default volume, range, and increment are
+export const MAX_KIOSK_VOLUME = 100;
+export const MIN_KIOSK_VOLUME = 28;
+export const MAX_VOLUME_INCREMENT = 10;
+export const MIN_VOLUME_INCREMENT = 1;
+export const INITIAL_VOLUME_INCREMENT = 3;
+export const VOLUME_INCREMENT_VALUE =
+  (MAX_KIOSK_VOLUME - MIN_KIOSK_VOLUME) /
+  (MAX_VOLUME_INCREMENT - MIN_VOLUME_INCREMENT);
+
+export function incrementToVolume(increment: number): number {
+  return (
+    MIN_KIOSK_VOLUME +
+    (increment - MIN_VOLUME_INCREMENT) * VOLUME_INCREMENT_VALUE
+  );
+}
+
+export class KioskTextToSpeech implements TextToSpeech {
+  private muted = false;
+  private volumeIncrement = INITIAL_VOLUME_INCREMENT;
+  private isVolumeIncreasing = true;
+
+  constructor() {
+    assert(window.kiosk, 'KioskTextToSpeech requires window.kiosk');
+  }
+
+  /**
+   * Directly triggers speech of text. Resolves when speaking is finished
+   * or cancelled.
+   */
+  async speak(text: string, { now = true }: SpeakOptions = {}): Promise<void> {
+    assert(now, 'method "speak"  with "options.now = false" is not supported');
```

Comment:
As @benadida mentioned in his earlier comment on this, I don't think we ever actually use `now = false`, so maybe we just strip it out altogether?